SPECS/conda/conda.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "conda-4.11.0.tar.gz": "1843355ccb93afb05f2a307e1fc37403fb9976da799236eebc3adee1c716c5fc",
-  "conda": "82512212b671e524a0f9fd26a2ba8e0b05fd55501c43160d63ed5c7d447f9fa4"
+  "conda-24.1.2.tar.gz": "6bc4f1f72a0edddefa10e0667d6ab2eb2f1956919a59508bf3bf6c001bf7a6e8",
```

Comment:
quite a big bump 🥇

Reply:
Yep, they recently switched to [calver](https://calver.org/). https://docs.conda.io/projects/conda/en/latest/release-notes.html#id98